include/zephyr/bluetooth/bluetooth.h

```diff
@@ -134,15 +134,15 @@ struct bt_le_per_adv_response_info {
 
 struct bt_le_ext_adv_cb {
 	/**
-	 * @brief The advertising set has finished sending adv data.
+	 * @brief The advertising set was disabled after reaching limit
 	 *
-	 * This callback notifies the application that the advertising set has
-	 * finished sending advertising data.
-	 * The advertising set can either have been stopped by a timeout or
-	 * because the specified number of advertising events has been reached.
+	 * This callback is invoked when the limit set in @ref
+	 * bt_le_ext_adv_start_param.timeout or @ref
+	 * bt_le_ext_adv_start_param.num_events is reached.
```

Comment:
```suggestion
	 * This callback is invoked when the limit set in 
	 * @ref bt_le_ext_adv_start_param.timeout or 
	 * @ref bt_le_ext_adv_start_param.num_events is reached.
```

Not a blocker, but I do prefer having the `@ref` on the same line as the value it references

Reply:
Good point. I will fix it.

I missed that because I used a generic line re-wrap tool. Wrapping text manually is not fun, so I think we should think about how this can be automated. Ideally clang-format or another tool would take care of formatting Doxygen too. But I have not seen anything that can do this job. That's why I think we should maybe start using the `#foo` instead of `@ref foo`, so that there is no white space. At least it's a point in favor of it.

Reply:
> That's why I think we should maybe start using the #foo instead of @ref foo, so that there is no white space. At least it's a point in favor of it.

I wouldn't be opposed to that, given the reasoning

Reply:
cc @kartben

Reply:
If you want to change style of referencing etc, it will be good to do that as a global change in seperate PR.